catalog/src/main/java/it/eng/catalog/rest/api/CatalogAPIController.java

```diff
@@ -11,7 +11,7 @@
 import org.springframework.web.bind.annotation.*;
 
 @RestController
-@RequestMapping(consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE, path = "/api/catalog")
+@RequestMapping(consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE, path = "/api/v1/catalogs")
```

Comment:
Should we use here also ApiEndpoints.***?
And in other API catalog controllers
This is case for negotiation api controller (could be used in tests and when changed to v2, one place change)